docker-compose.yml

```diff
@@ -20,14 +20,16 @@ services:
     restart: "no"
     env_file:
         - project.env
+    volumes:
```

Comment:
What is the purpose of this volume?

Reply:
This volume actually breaks the migration (see my comment for [V2___updated_schema.sql](https://github.com/petteri-te/github-exercise/pull/62/files#diff-1c057712e5624937804142b4886c5601a43c62c6471f3b32961912e56cbc7842))

Reply:
Hei @omilaeva , I have removed this volume as well from docker-compose.yml file. Take a look here: petteri-te/github-exercise@9a0e71b.